Strategies/IITB_KPIs.md

```diff
@@ -0,0 +1,74 @@
+
+The indicator will be measured by using [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) and monitor their total processing time from beginning to end. This amount, over time, will be monitored to indicate reduction in processing time.
+
+`rate of service = (New Processing Time - Old Processing Time) / Old Processing Time`
+
+## 3. **Reduction in toil of IITB and ESDC Services**
+
+Toil is the kind of work tied to running a production service that tends to be manual, repetitive, automatable, tactical devoid of enduring value, and that scales linearly as a service grows. Processing  time of ESDC services may decrease by increasing manual labour but that would defeat the purpose of digital.
+
+The indicator will be measured two ways:
+
+1. For toil in IITB Services, using ??? and measure the ratio of X over Y. 
+
+`ratio = X / Y`
+
+2. For toil in ESDC Services, using the [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) and measure the ratio of service requests to amount of personnel. 
+
+`ratio = service requests / service personnel`
+
+## 4. **Percentage of IITB services made available as SaaS**
+
+IITB services currently offered, as catalogued in the ITSM Repository, will be offered as SaaS for IITB consumers. This transformation to SaaS provisioning and maintenance will be measured by the rate at which those services are changed to self-served ones. The measurement will look at [IITB's Service Catalogue](http://iservice.prv/eng/service_catalogues/imit_services.shtml), which is being modernized for 2020, as reference point and monitor their transformation to SaaS.
+
+`percentage = SaaS / total ITSM services`
+
+## 5. **Availability uptime of ESDC core systems**
+
+Moving IITB into a service-oriented organization will require IITB to manage the complexity and growth of its services pro-actively. The measurement of availability is driven by time loss whereas the measurement of reliability is driven by the frequency and impact of failures. This indicator focuses on availability as it's what clients of the service are concerned with. The ***service's*** maximum allowable downtime, as agreed by the BCP Business Impact Analysis exercise, will be used to assess IITB's service availability. The Core systems will be the ones flagged as *high business value* within [ESDC's Application Portfolio Management (APM) program](http://dialogue/grp/PR6303013/APM_Program/default.aspx).
+
+`service availability = (total elapsed time - sum of downtime) / total elapsed time`
+
+*Note:* Services are never expected to be up 100% of the time. [See here](http://www.gcpedia.gc.ca/wiki/OCIO_Application_Portfolio_Management/Application_Portfolio_Management_User_Guide#Application_Mission_Criticality_and_Critical_Services) for more details.  
+
+## 6. **Increase rate of project intake by IITB**
```

Comment:
I don't know about project intake or even project completion/success rate. It may be out of our control as well and if we're successful, might accelerate or decelerate? It may turn into a product management culture instead and as such there would be less projects, more continuous development/improvement?

Reply:
Good point re Product Management.
Currently, IITB cannot meet demands. If you have a "project" you'll want to engage IITB with, they'll probably say "we have time for you in 2024". Our strategies will be to improve this demand fulfillment. Through initiatives like (for example) moving to Product Management, DevOps, Bi-Modal Governance, API-first design 

Would there be another metric that we can use to measure IITB's ability to improve its capacity? Even with product mgt and CD, there must be something tracked that we can use. Thoughts?

Reply:
Gartner must have something :wink:

Adding in a separate issue to discuss.